docs/guide/http_cache.md

```diff
@@ -494,8 +494,11 @@ Here are some generic recommendations on how to approach personalized content wi
 1\. ESI + Vary by Cookie:
 
 - As default VCL strips everything but session cookie this means this will effectively be cached "per user".
-   - If you are on single server setup with neither Varnish nor Fastly, you can do the same cookie logic on the Web server instead.
+    - If you are on single server setup with neither Varnish nor Fastly, you can do the same cookie logic on the Web server instead.
```

Comment:
I am a bit fuzzed by this sentence... We are in chapter "ESI + Vary by Cookie", so you obviously are using Varnish or Fastly. And if you are not, you're likely using the Symfony proxy so you cannot simply do logic on the web server... ?

Reply:
![Screenshot 2020-10-13 at 09 42 45 ](https://user-images.githubusercontent.com/289757/95831308-292c5400-0d39-11eb-8592-5f8f513c10d5.png)


This section is for all supported proxies, except proposals below which depends on being able to modify VCL logic to happen.